external-secrets/install-external-secrets-operator.hbs.md

```diff
@@ -65,11 +70,55 @@ To install External Secrets Operator:
    Added installed package 'external-secrets'
    ```
 
+   *OPTIONAL* The YAML values file specifies optional configuration for the
+   External Secrets Operator.  At this time there is only one setting that can
+   be used to configure the External Secrets Operator:
+
+     kubernetes_distribution: "" or "openshift"
+
+   If you want to run External Secrets Operator on an OpenShift cluster then you
+   must set the `kubernetes_distribution` value to `openshift`.  Using this
+   setting removes some of the default security settings in External Secrets
+   Operator so that OpenShift can replace the security settings with its own.
+
+   If you are running the External Secrets Operator on any other platforms than
+   you can leave `kubernetes_distribution` as an empty string (i.e.: `""`) or
+   omit the value entirely.
+
+   The External Secrets Operator should be able to inherit this setting from the
+   `shared.kubernetes_distribution` in the top-level values file applied to the
+   TAP install itself.
+
```

Comment:
```suggestion

```
Suggestion: Please remove this section of the text. External Secrets Operator is a pre-req for TAP gitops install.